wurst/extensions/UnitExtensions.wurst

```diff
@@ -105,3 +105,22 @@ public function unit.dropItems(int count)
 
 public function unit.dropItems()
     this.dropItems(this.inventorySize())
+
+public function unit.setUserDataEx(int data)
+    SetUnitUserDataEx(this, data)
+
+public function unit.getUserDataEx() returns int
+    return GetUnitUserDataEx(this)
+
+public function unit.addSleep(bool value)
+    UnitAddSleep(this, value)
+
+public function unit.setCreepGuard(bool value)
+    SetUnitCreepGuard(this, value)
+
+public function unit.isImmobilized() returns bool
+    return (
+        this.getAbilityLevel('Beng') > 0 or

```

Comment:
Having comments explaining these IDs would be good, at least until we can reference them by variable name.